test/e2e/specs/domain-upsell/domain-upsell__sidebar.ts

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
Great test @lupus2k, it's working well and validates the whole flow!

Reply:
Mi next plan is to test this flow with the experiment